Final touches desktop (For Code Review)

AmineHLub opened the pull request:
- Fixed minor left issues with the bottom header including margins and styling using flexbox to center
- Responsive @ <768px for mobile and >768px for Desktop view

AmineHLub committed:
Fix bottom header layout

ADD background images at the bottom

AmineHLub committed:
Fix bottom side margin (was too close)

AmineHLub committed:
Fix mobile background bottom frame being mispositioned

AmineHLub committed:
fix Ipad users view

SajjadAhmad14 commented on style.css:
```diff
@@ -51,7 +51,7 @@
+    background-position: 100% 40rem, 100% 17rem, 100% 3rem, 50% 3rem, 0% 32rem, 30% 33.5rem, 5% 11rem, 100% 97%, 50% 99.8%;
   }
 
   h1 {
```
Please use border color for cards as defined by design. Kindly see the screenshot below also `see project` button height should be a bit small.
![cards](https://user-images.githubusercontent.com/35504149/129953886-f8b57216-f5f7-426d-a5a1-82cddc7fbe6e.png)

SajjadAhmad14 commented on style.css:
```diff
@@ -51,7 +51,7 @@
     background-repeat: no-repeat;
-    background-position: 100% 40rem, 100% 17rem, 100% 3rem, 50% 3rem, 0% 32rem, 30% 33.5rem, 5% 11rem, 100% 300rem, 50% 99.8%;
+    background-position: 100% 40rem, 100% 17rem, 100% 3rem, 50% 3rem, 0% 32rem, 30% 33.5rem, 5% 11rem, 100% 97%, 50% 99.8%;
   }
```
Please note the height of the `Get my Resume` button height in design and fix yours accordingly.
![resume-btn](https://user-images.githubusercontent.com/35504149/129954752-9c6fa65d-418e-4f09-b22a-835a6c83ddc7.png)

SajjadAhmad14 commented on more-style/desktop-contact.css:
```diff
@@ -14,7 +14,11 @@
   }
 
```
The contact section on mobile view has a background image on the right side, but the project does not have one. Please add a background image.

SajjadAhmad14 reviewed (changes requested):
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Beretta Hajltaief, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
- [ ] GitHub flow is used properly.
- [ ] README is professional.
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

AmineHLub committed:
Fix Buttons size required

Fix border Color

Fix background images

Fix another found issue with the Get my resume button (was not centered in the mobile version)

karvel-code reviewed (approved):
Hi @AmineHLub , 

## Status Approved!!  🎉  🎉 
You have worked on the changes requested by the previous reviewer and hence your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
